frame-server/server/profile/lib/businessRules.js

```diff
@@ -33,13 +34,13 @@ var businessRules = {};
         console.log("Account found using account id");
         console.log(account);
 
-        profileData = {
```

Comment:
Any reason of redeclaring this here after line 25?

Reply:
Should have removed the declaration at line 25.

Reply:
done 👍